config/local.yaml

```diff
@@ -5,6 +5,7 @@ leaderElection:
 interval: "5m"
 storagePath: /tmp/insights-operator
 endpoint: http://[::1]:8081
+conditionalGathererEndpoint: https://console.redhat.com/api/gathering/gathering_rules
```

Comment:
I am not sure we need it here, but what about [here](https://github.com/openshift/insights-operator/blob/master/pkg/cmd/start/start.go#L27)? Did you test it in a container please?